src/common/tests/CommonUtilsUT.cpp

```diff
@@ -236,17 +241,20 @@ TEST_F(CommonUtilsTest, ExecuteCommandWithStdErrOutput)
     char* textResult = nullptr;
 
     EXPECT_EQ(127, ExecuteCommand(nullptr, "hh", false, true, 100, 0, &textResult, nullptr, nullptr));
-    EXPECT_NE(nullptr, strstr(textResult, "sh: 1: hh: not found\n"));
+    EXPECT_NE(nullptr, strstr(textResult, IsMariner() ? "sh: line 1: hh: command not found" : "sh: 1: hh: not found"));
 
     if (nullptr != textResult)
     {
         free(textResult);
     }
```

Comment:
Replace this entire block (lines 246 - 249) with a single FREE_MEMORY(textResult)